konami/products.md

```diff
@@ -550,6 +559,7 @@ chronologic order, e.g. regarding release dates.
 * `SPU`: GRANDCROSS LEGEND  `#SCRAPE:konami:eagate`
 * `SS4`: FEATURE Premium 7・8弾  `#SCRAPE:konami:eagate`
 * `T44`: jubeat (新筐体) [unreleased]  `#WEB:https://game.mirai-media.net/jubeat-jaepo2020/`
+  * `GQT44-JA`
```

Comment:
I don't think dongles or manuals for this exist to verify this...

Reply:
I forgot where I seen this, but I'll remove it